tests/fixtures/factories.py

```diff
@@ -0,0 +1,171 @@
+from faker import Faker
+from polyfactory import Use
+from polyfactory.decorators import post_generated
+from polyfactory.factories.pydantic_factory import ModelFactory
+
+from ref_builder.models import MolType
+from ref_builder.utils import Accession
+from ref_builder.ncbi.models import NCBIGenbank, NCBISource, NCBISourceMolType
+from tests.fixtures.providers import (
+    AccessionProvider, OrganismProvider, SequenceProvider, SourceProvider
+)
+
+DNA_MOLTYPES = {
+    NCBISourceMolType.GENOMIC_DNA,
+    NCBISourceMolType.OTHER_DNA,
+    NCBISourceMolType.UNASSIGNED_DNA
+}
```

Comment:
Docstring. Can just be a list.

Reply:
We don't iterate over values in there so I think a set will be fine.